### DISCUSSION TITLE
Discussion about allocation of 0.54% of $pink Total Supply, which is for Darwinia

### DISCUSSION THREAD

**Discussion by Laki389** [darwinia-network/darwinia#1403] [General]
![1](https://github.com/darwinia-network/darwinia/assets/133054568/b7be935a-2244-4e8e-b3a8-3229ba77c1db)

1. **Brief introduction.**

Hi, I think it's time to start a public debate on the principles/how to allocate the 0.54% of $pink Total Supply that will be sent to Darwinia.

$pink has billed itself as "the first Polkadot meme project", something that will indeed be the case as it is expected that TGE will happen before $ded.

It started as a light, funny, simple, addictive, retro game, accessible to everyone, played by developers, influencers, people from various Polkadot ecosystem projects (and beyond).
The project has gained significant attention, it is possible that the part intended for Darwinia will be worth a significant amount of dollars in the future, so I would ask that we approach this seriously and I hope for participation in the discussion from everyone (Itering developers, Managers, Ambassadors, active members of the community, stakers, Helix bridge (team and users), Snowswap (team and users) and everyone I forgot.

2. **Info, project data.**

- A bit more context on PINK: 
![2](https://github.com/darwinia-network/darwinia/assets/133054568/050ea594-9737-4a19-abce-89c6f955759f)

- PINK Twitter: https://twitter.com/pinkonomic

- PINK`s Pinkonomics
![photo_2023-12-22_16-54-53](https://github.com/darwinia-network/darwinia/assets/133054568/4946fcd9-9883-422c-a823-656699d7a2ca)

- Pink website: https://dotispink.xyz/

- Pink Paper: https://dotispink.xyz/paper  (Darwinia included)

- Pink Points for Parachains (0.54% Total Supply for Darwinia)
![GEiraN8XYAAHF4X](https://github.com/darwinia-network/darwinia/assets/133054568/b3d53448-a157-4b2b-9e50-ce3310e92fd4)

- PINK TG: https://t.me/PINKonomic

- Welcome to Darwinia. https://twitter.com/pinkonomic/status/1749837208041746833

3. **The way other projects plan to do the allocation.** 
(the list of links will be updated)

- Moonbeam: 
https://twitter.com/MoonbeamNetwork/status/1752051502968127706
https://forum.moonbeam.network/t/we-are-getting-pink-and-here-is-how-to-get-your-share/1462

- Bifrost:
https://twitter.com/BifrostFinance/status/1750831048047444179

**Comment by danairey** [comment]
Really a tough one, incentivize people that use one of Darwinia products like helix is the most appropriate. Otherwise darwinia needs funds to market its product and get devs to build . Sell it all and use the funds to actually grow darwinia dapps. Lets worry about growing our products rather.

**Reply by Green-Chiles** [reply]
I think it would be a mistake for the team just to dump the PINK tokens to add cash to their coffers. In effect, you would be dumping on the entire Polkadot community, and this is supposed to be a Polkadot wide community event. Not sharing with Darwinia supporters at all would be bad.  Simply dumping the tokens as a project would be even worse.

**Comment by Green-Chiles** [comment]
My 2 cents:  among other things, some allocation for stakers, community mods and long term community supporters. 

Crowdloan support probably isn’t the best. Too many people who just came for tokens to dump and then left.

**Comment by 2xJDubs** [comment]
To start I am glad the Darwinia joined in on the fun and gained an allocation,  I think that as Laki said we need to be smart about distribution.  

First off I would like to see a portion of the allocation kept for the treasury for later usage that can be determined by governance, beauty of Darwinia now using OpenGov is anyone can suggest a usage.

I think a distribution among team members, ambassadors - past and present, and advocates makes sense.

The next tier should be active community members and OG stakers, which I define as people that locked their RING for 3 plus years 2021 or before.

We should also reward any new stakers that locked RING for 3 years, so people who staked in 2022 or later.

I also some of the allocation should be used to incentivize of community activity and could be included in rewards for contests and the weekly quizzes.

I am not sure how best to set the distribution percentages but I would start with the Treasury having between 25% and 50%.  Again this can be distributed later but don't think it should be sold to fund anything.  Incentivizing out community with $PINK will help to engage the community and attract attention that will be the best marketing we can get.

**Comment by SasoLithops** [comment]
Great job Laki in presenting this.

I also think this funds should be intented to promote Darwinia, Pink and Polkadot. 

I especialy agree with giving some to active community members and what Jesse called OG stakers(anyone locking RING for three years). 

I would propose:

Team members,ambassadors,advocates 15%
OG stakers 15%
Active community members 15%
Other community activity(quizes,games...) 15%

The rest (40%) we leave and give(some %) also as random airdrops for Darwinia governance voters when OPENGOV is live.

**Comment by SasoLithops** [comment]
Dear community!

After initial discussion above  I propose we put ideas together and vote on best. This will last for two weeks or more if community desires.

Next rules apply(let me know if you disagree with any or want me to add more)

1. Proposal must specify what % of the total PINK treasury ammount should go to what.
2. Please explain your decision behind choosing certain %, not just randomly post one.
3. PLEASE put in **bold** what you think should go on temperature check proposal. I will then post only that for a one week temperature check proposal for some extra finishing touches opinions. Then I will put proposal live for all of us Darwinia community to vote on.
4. Pleas write in mind for community gains not personal gains

I look forward to your participation. MUCH APPRECIATED.


Let me give example:

**Proposal #1**

**Team members, ambassadors, advocates that are and have been with Darwinia 15%**

_**Most of this members didn't have time to play games. It is like allocating 15% of token issued to team members for hard work.**_

**OG Stakers 15%**

**_The ones that have Ring locked for one or more years._**
**_1 year-2 years: certain ammount_**
**_2 years-3 years: twice the certain ammount_** 

**Incentives for KTON/RING Liquidity providers on Snowswap.xyz 30%**

**_I would suggest running this incentives for keeping liquidity for 3 months, 6 months or 12 months. The more you hold the more allocation you get. We just need to think first how to get to cheaper KTON then bridging it from Ethereum network( Thank you Jesse for pointing this out)_**

**The rest 40% go to treasury for time being**

**Comment by saoud1976** [comment]
For me the best way is to reward darwinia staker for collobaration and implication in the network

**Comment by Laki389** [comment]
1. **Introduction.**
In compiling my proposal, I was guided by the principles of simplicity and practicality.

2. **Proposal for distribution.**

-  **$ring staking or in deposits (locked):** **35%**

Snapshot time, **October 11, 2023**, the day $ring traded at its lowest. (according to Coingecko)

**Method of distribution**, according to the amount of $ring that was staked or in deposits.
For example, wallets with less than 10 000 $ring, wallets with 10 000 - 50 000 $ring, wallets with 50 001 - 100 000 $ring, wallets with 100 001 - 200 000 $ring; 200 001 - 500 000 $ring; 500 001 $ring - 1 000 000 ring; 1 000 001 $ring - 2 000 000 $ring; 2 000 001 $ring - 5 000 000 $ring; more than 5 000 000 $ring.
This is just an example.

- **Kton staking: 20%**

Snapshot time, **September 26, 2023**, the day $kton traded at its lowest. (according to Coingecko)

**Allocation method**, according to the amount of staked Kton on the wallet.
Less than 1;
1.01 - 5;
5.01 - 10;
10.01 - 20
20.01 - 50
50.01 - 100
100.01 - 200
200.01 - 500
500.01 - 1000
1000.01 - 2000
2000.01 - 5000
over 5000.
Also, just as an example.

-   **5% distribution** to all who have merit for:

 

- [ ] first gave the idea for Darwinia to participate in PINKDROP

 

- [ ] participated in introducing PINK to the Itering team (via Discord, Github) and with guidance on what to do.

 

- [ ] as a developer, participated in Register PINK on Darwinia

 

- [ ] everything that follows until the final distribution, for the allocated time and effort.

- **40% remains in the Treasury.**
In the future, if projects like Snowswap and Helix want to participate, a team member will have to make and present a plan, which we as a community will decide on.
This is the case in other Polkadot ecosystem projects.
Except for projects that use Darwinia technology, part of these funds can be for any creative solution and idea.

**Reply by 2xJDubs** [reply]
I think this is a brilliant plan and really don't have anything that could make it better.  Great proposal Laki.

Edit: After some more thought, I do have one thing I would like to add to Laki's proposal.  I think that people who were staked at these two times still need to be staked with at least 2/3s of the amount to be eligible.

**Reply by SasoLithops** [reply]
Laki I like your idea more then mine😁

Only thing I would lower percentage and also incentivise liquidity providers for KTON/RING on Snowswap too. Othervise only small ammount would be left in treasury if we also incentivise liquidity providers.

But definitely a great option to vote on.

**Reply by Laki389** [reply]
> Edit: After some more thought, I do have one thing I would like to add to Laki's proposal. I think that people who were staked at these two times still need to be staked with at least 2/3s of the amount to be eligible.

@2xJDubs 
As I wrote, my proposal is guided by the principle of simplicity, the additional condition complicates it. (meaning it could potentially require more manual work).

If you insist on this condition, and if gathering that data and cross-referencing it is an easy task for Xavier, then so be it.

When writing the proposal, I was guided by the fact that it should be as simple as possible, since we are not developers, but I suggest that we consult with Xavier.

**Reply by Laki389** [reply]
> Only thing I would lower percentage and also incentivise liquidity providers for KTON/RING on Snowswap too. Othervise only small ammount would be left in treasury if we also incentivise liquidity providers.

@SasoLithops 

I think it is better that the PINK section for Darwinia is shared with as many community members as possible.
At the same time, care should be taken that the parts intended for staking/deposit on Ring and staking on KTON are not insignificant.
We need to consider the number of addresses.

When we consider the previously mentioned 2 things, I think the percentages are fine.

About Snowswap, I think we should follow the example of other parachains.
The Snowswap team should apply themselves with a plan to allocate their share.
I think that in that 40% at Treasury there is enough for Helix, Snowswap, potential other projects or all kinds of creative ideas.

In short, Snowswap LP would be a dozen people and ring staking/deposit and kton staking are hundreds or thousands of people.
Let's spread the idea of PINK to as many people as possible, but at the same time, let's make sure that these are not insignificant amounts.
Like for example with Moonbeam/GLMR, because of the much larger number of addresses they have.
So, with the 40% share in Treasury, I was also thinking of Snowswap.

**Comment by 2xJDubs** [comment]
> > Edit: After some more thought, I do have one thing I would like to add to Laki's proposal. I think that people who were staked at these two times still need to be staked with at least 2/3s of the amount to be eligible.
> 
> @2xJDubs As I wrote, my proposal is guided by the principle of simplicity, the additional condition complicates it. (meaning it could potentially require more manual work).
> 
> If you insist on this condition, and if gathering that data and cross-referencing it is an easy task for Xavier, then so be it.
> 
> When writing the proposal, I was guided by the fact that it should be as simple as possible, since we are not developers, but I suggest that we consult with Xavier.

@Laki389, completely understand your approach and agree having the extra requirement would add some effort.  I would just hate to reward someone that happened to unbond the day after the snapshot date and sold everything.  Maybe we would just compare a list from the snapshot dates to a current staker list which should not be too difficult.  If we get the lists in excel I can make short work of the comparison.

**Reply by SasoLithops** [reply]
Maybe we could introduce double/triple date snapshot. Those who staked on all three dates get tokens.

**Reply by Laki389** [reply]
> @Laki389, completely understand your approach and agree having the extra requirement would add some effort. **I would just hate to reward someone that happened to unbond the day after the snapshot date and sold everything.** Maybe we would just compare a list from the snapshot dates to a current staker list which should not be too difficult. If we get the lists in excel I can make short work of the comparison.

@2xJDubs 
Snapshot date is at the bottom, if they sold the day after, they certainly didn't get rich :)

But in the interest of time, since I think we are late compared to other parachains, I agree to slightly modify my proposal. 
(if elected of course)
Since I know @SasoLithops  actively follows Cosmos, he could give some ideas/experiences how they do it there.

**Reply by SasoLithops** [reply]
I am all for Laki's proposal. As for dates, they are usually well thought out and have something to do with project launching.We are not new project so hard to compare. Also they have minimum and maximum cap.
Not sure we would like to complicate with that as I am trying as Laki said keep it simple.
Not sure if we can get data of stakers from Darwinia 1.0 with their current EVM ADDRESS not substrate one as we would send Pink to EVM. If we could get that data I would reward the ones that stuck with Darwinia when UST collapsed and FTX affair was in. Need to check exact date.And the other date would be like you proposed Laki. It is good.
What do you think?

**Reply by Laki389** [reply]
Considering the suggestions here and on TG, I've decided that my proposal is as written.

If anyone has any suggestions on my proposal, feel free to use parts of it and modify it and let it be their proposal.
Let his/her proposal be detailed and thought out and have answers to all the problems and questions that may arise from having 2 or 3 Snapshot dates and so on.

In my proposal, only the levels/tiers are written as an example, I will make a definitive proposal for that as well, after Xavier gives us data on the intersection of Ring and Kton.
Of course, if my proposal is voted on first.

**Comment by 2xJDubs** [comment]
I think we are all in agreement that Laki's proposal is the best for distribution.  How do we move forward from here?  Does this need to go to an OpenGov vote?

**Reply by Laki389** [reply]
I think we should first formally choose the best proposal here and then consult with Denny/Xavier on what to do next.

**Comment by SasoLithops** [comment]
![Screenshot_2024-02-29-10-25-04-622_com.android.chrome.jpg](https://github.com/darwinia-network/darwinia/assets/101176902/fcf64c86-2e4c-4257-937c-7d7eda007463)

I think we should set a max cap too.

**Comment by Laki389** [comment]
> Dear community!
> 
> After initial discussion above I propose we put ideas together and vote on best. This will last for two weeks or more if community desires.
> 
> Next rules apply(let me know if you disagree with any or want me to add more)
> 
>     1. Proposal must specify what % of the total PINK treasury ammount should go to what.
> 
>     2. Please explain your decision behind choosing certain %, not just randomly post one.
> 
>     3. PLEASE put in **bold** what you think should go on temperature check proposal. I will then post only that for a one week temperature check proposal for some extra finishing touches opinions. Then I will put proposal live for all of us Darwinia community to vote on.
> 
>     4. Pleas write in mind for community gains not personal gains
> 
> 
> I look forward to your participation. MUCH APPRECIATED.
> 


Hi @SasoLithops , would you like to lead the process?

How much more time can proposals be given?

When will we go to the vote?

**Reply by SasoLithops** [reply]
5 more days left.

I think we all agree on your proposal. So this one is the one that should go on vote.
We just need now to get data on that days @AurevoirXavier will provide help with that.
Then we can discuss the criteria for Ring and Kton.
I have looked into more airdrops and am more for 1 Ring=x Pink, as setting different rangs could lead into complications as for example between 10000 and 100000 could be 500 stakers and between 100000 to 500000 20 stakers. So how to distribute then? Would be harder to calculate.
Also depending on whale staking total value. If it is above 25% I would set a whale cap. This number is up to discussion.

**Comment by Laki389** [comment]
1. **Introduction.**

After the debate we had in the old ambassadorial TG, I slightly modified my original proposal:

- 2 snapshot dates have been introduced.

- In the distribution method, instead of different levels, the principle of 1 $ring = x (unknown) amount of $pink is introduced. That amount will become known later, after Xavier helps us find out the total amount of staked+deposited $ring on the snapshot date below.
The same goes for $kton.


2. **Distribution proposal.**

- **$ring staking or in deposits (locked): 35%**

 Snapshot dates:
October 11, 2023, the day $ring traded at its lowest. (according to Coingecko) ; PINK **eligibility** is acquired on this day.

23rd January 2024, 16:51 UTC - the moment when the Pink twitter account welcomes Darwinia and assigns her 0.54% total supply. ; **according to the data of this day, it is determined which address - how much $pink will receive.** https://twitter.com/pinkonomic/status/1749837208041746833

**Distribution method**, 1 $ring = x  $pink.

- **Kton staking: 20%**

 Snapshot dates:
September 26, 2023, the day $kton traded at its lowest. (according to Coingecko) ; PINK **eligibility** is acquired on this day.

23rd January 2024, 16:51 UTC - the moment when the Pink twitter account welcomes Darwinia and assigns her 0.54% total supply. ; **according to the data of this day, it is determined which address - how much $pink will receive**. https://twitter.com/pinkonomic/status/1749837208041746833

**Distribution method**, 1 $kton = x $pink.

- **5% distribution** to all who have merit for:

- [ ] first gave the idea for Darwinia to participate in PINKDROP

- [ ] participated in introducing PINK to the Itering team (via Discord, Github) and with guidance on what to do.

- [ ] as a developer, participated in Register PINK on Darwinia

- [ ] everything that follows until the final distribution, for the allocated time and effort.

- **40% remains in the Treasury**.
In the future, if projects like Snowswap and Helix want to participate, a team member will have to make and present a plan, which we as a community will decide on.
This is the case in other Polkadot ecosystem projects.
Except for projects that use Darwinia technology, part of these funds can be for any creative solution and idea.

**Reply by 2xJDubs** [reply]
I am 100% on board with this path forward.

**Reply by SasoLithops** [reply]
Great Job finalising this Laki. Absolutely Yes from me.

**Reply by JuneCribb** [reply]
A Yes from me too. 👏

**Comment by sonoferin2000** [comment]
how about an alocation to holders of darwinia polkapet  nfts?

**Reply by 2xJDubs** [reply]
I think this is something we can consider with a proposal to spend some of the 40% going to the treasury.

**Comment by 2xJDubs** [comment]
For community clarity and before distribution occurs the following methodology was used to determine how much PINK was dropped to each account:

35% of the PINK for RING staking/deposits is given based on the following:
(Amount of RING Staked + Amount of RING in Deposit) * (35% PINK RING allocation / (Total amount of RING staked and deposited))

20% of the PINK for KTON staking is given based on the following:
Amount of KTON Staked * (20% PINK KTON allocation / Total amount of KTON staked)

The ratio of PINK per RING is 0.0193264190024495

The ratio of PINK to KTON is 41.4980010142839
